JavaDirectory/src/main/java/com/bug/free/invention/api/controllers/DBConfig.java

```
@@ -0,0 +1,26 @@
+package com.bug.free.invention.api.controllers;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DBConfig {
+
+    private  static String Username = "";
+    private static String Password = "" ;
+    private static String host = "academy2020.cpc8rvmbbd9k.eu-west-2.rds.amazonaws.com";
+
+    public DBConfig(){
+
+    }
+
+    public static Connection getConnection(){
+        try{
+            Connection conn = DriverManager.getConnection("jdbc:mysql://"+ host + "/teamCteam_Josh?useSSL=false", Username, Password);
+            return conn;
+        } catch (SQLException throwables) {
+            throwables.printStackTrace();
+            return null;
```

By returning `null` you're triggering a `NullPointerException` from the consuming code, which may make it harder to find out what went wrong when it fails. If you want to print the stack trace (dubious anyway) then instead of returning null, rethrow the same exception. Also `throwables` is a very bad name for an exception.